js/schedule.js

```diff
@@ -670,6 +688,44 @@ function Schedule(options) {
             }
         });
 
+        // toggle individual schedule blocks on header tap
+        schedule.$container.on('click', '.slider-control', function(e) {
+            var clicked = $(this);
+            var targetBlock = clicked.next('.page-block');
+            targetBlock.toggleClass('closed');
+            if (targetBlock.hasClass('closed')) {
```

Comment:
yeah, I thought about that after I wrote the same thing twice, but then I was super tired and I went to bed :)

will extract and push an update